lte/gateway/python/magma/mobilityd/rpc_servicer.py

```diff
@@ -164,14 +163,14 @@ def ListAllocatedIPs(self, ipblock_msg, context):
 
     def AllocateIPAddress(self, request, context):
         """ Allocate an IP address from the free IP pool """
```

Comment:
we shld also add support to add IPv4 and v6 address in single call to avoid multiple gRPC call from MME in such cases.